doc/STYLE_GUIDE.md

```diff
@@ -6,8 +6,7 @@
     topic the document will describe (e.g., `child_process`).
   * Filenames should be **lowercase**.
   * Some files, such as top-level markdown files, are exceptions.
-  * Older files may use the `.markdown` extension. These may be ported to `.md`
-    at will. **Prefer `.md` for all new documents.**
+  * Use `.md` as the file extension for markdown files.
```

Comment:
Is this necessary given the extension is included in the example in the parent bullet (`lowercase-with-dashes.md`)?

Reply:
Or maybe merge with the existing bullet about filenames should be lowercase above.

Reply:
I'm happy to remove it entirely. It seems unlikely that we will have a rampant problem with people using the wrong file extension for new docs.

Reply:
+1 for removing entirely